### PR TITLE
[#6] Holdings autodoc upload

### LINKED ISSUE #6: Automate autodoc publishing
# Clarification and motivation
We have a mechanism for building and uploading documentation for the contracts. Let's use it
here as well.

# Acceptance criteria
We have automatically uploaded autodoc for contracts in this repository

### PR DESCRIPTION
**Pull request by rvem**
## Description

Added autodoc publishing for Holdings contract.

Sample documentation can be found [here](https://github.com/serokell/tezos-globacap/blob/autodoc/rvem/%236-autodoc-upload/Holdings-contract.md).

## Related issue(s)

Resolves #6

#### Related changes (conditional)

- Tests
  - [x] If I added new functionality, I added tests covering it.
  - [x] If I fixed a bug, I added a regression test to prevent the bug from
        silently reappearing again.

- Documentation
  - [x] I checked whether I should update the docs and did so if necessary:
    - [README](../tree/master/README.md)
    - Haddock

#### Stylistic guide (mandatory)

- [x] My commits comply with [the policy used in Serokell](https://www.notion.so/serokell/Where-and-how-to-commit-your-work-58f8973a4b3142c8abbd2e6fd5b3a08e).
- [x] My code complies with the [style guide](../tree/master/docs/code-style.md).

### CODE REVIEW

File: .buildkite/pipeline.yml [hunk 1]
```diff
@@ -72,7 +72,7 @@ steps:
       else CONTRACT_DOC_STEP="contract doc release";
       fi
     - buildkite-agent artifact download Holdings-contract.md tmp/
-        --step "$CONTRACT_DOC_STEP"
+        --step "$$CONTRACT_DOC_STEP"
```
Comment (gromakovsky) [inline .buildkite/pipeline.yml:75]: How does it work? What syntax is this?

Comment (gromakovsky) [reply .buildkite/pipeline.yml:75]: Why two dollars?

File: README.md [hunk 2]
```diff
@@ -13,6 +13,10 @@ Tezos Globacap project provides two entities:
 
 There is a [specification](/docs/specification.md) for the required smart contracts.
 
+## Holdings contract documentation
+
+Holdings contract documentation can be found [here](https://github.com/serokell/tezos-globacap/-/blob/autodoc/master/Holdings-contract.md).
```
Comment (gromakovsky) [inline README.md:18]: I would make the difference between autodoc and `specification.md` a bit clearer, as we did it in another repo (a private one).

File: .buildkite/pipeline.yml (hunk 1 above)
Comment (rvem) [reply .buildkite/pipeline.yml:75]: Apparently, buildkite eats the first dollar and the second dollar eats shell

Comment (gromakovsky) [reply .buildkite/pipeline.yml:75]: Bon appetite to both of them.

File: README.md (hunk 2 above)
Comment (rvem) [reply README.md]: updated, btw

**Pull request merged**